File: sdklibrary/src/main/java/com/example/sdklibrary/mvp/model/ApiResponse.java
package com.example.sdklibrary.mvp.model;

/**
 * Date:2023-01-05
 * Time:15:24
 * author:colin
 */
public class ApiResponse<T> {

    private int ts;
    private int code;
    private String msg;
    private T data;

    public int getTs() {
        return ts;
    }

    public void setTs(int ts) {
        this.ts = ts;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }
}

File: sdklibrary_kt/src/main/java/com/sdk/sdklibrary/mvp/model/ApiResponse.kt
package com.sdk.sdklibrary.mvp.model

/**
 * Date:2023-01-05
 * Time:15:24
 * author:colin
 */
class ApiResponse<T> {
     var ts = 0
     var code = 0
     var msg: String? = null
     var data: T? = null
}